src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/BundleException.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.em.orchestrator.automatedbundling;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BundleException extends Exception {
```

Comment:
can we make it runtime exception so oyu do not need to add it method signatures?